src/ILVerify/tests/TestDataLoader.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Internal.IL;
+using Internal.TypeSystem.Ecma;
+
+namespace ILVerify.Tests
+{
+    /// <summary>
+    /// Helper method to load the binaries generated based on the il code, which drive the tests
+    /// </summary>
+    static class TestDataLoader
+    {
+        /// <summary>
+        /// The folder which contains the core assembly (containing System.Object, etc.)
+        /// </summary>
+        static string COREASSEMBLYPATH = @"..\..\..\..\..\..\bin\Product\Windows_NT.x64.Debug\CoreTestAssembly\\";
+
+        /// <summary>
+        /// The folter with the binaries which are compiled from the test driver IL Code
+        /// </summary>
+        static string TESTASSEMBLYPATH = @"..\..\..\ILTests\";
```

Comment:
Could you please share more about this? Currently there are only two projects: 1 ILVerify, 2 ILVerify.Tests, which contains the xunit code, and the .il file is just in a subfolder. (to get the dlls, which we test on I call ilasm manually). Should I move the .il files into a separate project and do the referencing trick with that project? I’m trying to do that, but I have problems with creating the .ilproj based project. I did not find that in VS, so I tried to do it manually (based on src\ILCompiler.TypeSystem\tests\ILTestAssembly\ILTestAssembly.ilproj), but VS cannot open the project I created manually (ILVerify.Tests.ILTests.ilproj : error  : Project file is incomplete. Expected imports are missing.). So is this the way I should do it? If yes, is there a documentation on .ilproj? (google does not really help..).